### ISSUE TITLE
firebase.appdistribution plugin 4.1.0 not available

### ISSUE THREAD

**Issue by fleficher** [firebase/firebase-android-sdk#5703]
### [REQUIRED] Step 2: Describe your environment

  * Android Studio version: Hedgehog 2023.1.1 Patch 2
  * Firebase Component: App Distribution Gradle Plugin
  * Component version: 4.1.0

#### Steps to reproduce:

In build.gradle.kts file:

```

plugins {
    // ...
    id("com.google.firebase.firebase.appdistribution") version "4.1.0" apply false
}
```
Produces the following error:
```
* Exception is:
org.gradle.api.plugins.UnknownPluginException: Plugin [id: 'com.google.firebase.firebase.appdistribution', version: '4.1.0', apply: false] was not found in any of the following sources:

- Gradle Core Plugins (plugin is not in 'org.gradle' namespace)
- Included Builds (None of the included builds contain this plugin)
- Plugin Repositories (could not resolve plugin artifact 'com.google.firebase.firebase.appdistribution:com.google.firebase.firebase.appdistribution.gradle.plugin:4.1.0')
  Searched in the following repositories:
    Google
    MavenRepo
    Gradle Central Plugin Repository
    maven(https://maven.pkg.github.com/BackMarket/*)
```

**Comment by lehcar09** [comment]
Hi @fleficher , thank you for reaching out and reporting the issue. I was able to reproduce the issue you’re experiencing. Per checking, I see that the artifact **"com.google.firebase.appdistribution"** version 4.1.0 plugin does not exist in the google and maven repository. However, I see that the version 4.1.0 artifact for build script dependency **"com.google.firebase:firebase-appdistribution-gradle"** exists in the Google repository.

> build.gradle.kts
```
buildscript {
    dependencies {
        classpath("com.google.firebase:firebase-appdistribution-gradle:4.1.0")
    }
}
```

I’ll notify our engineers about this and see what we can do here.

**Comment by lehcar09** [comment]
Hi @fleficher, apologies for the inconvenience. According to our engineers, this was a mistake on our side. We pushed com.google.firebase:firebase-appdistribution-gradle but didn't push com.google.firebase.appdistribution:com.google.firebase.appdistribution.gradle.plugin. 

The missing artifact has been pushed and I was able to confirm that the issue has been resolved. Let me know if there's any issue.
